kebechet/managers/update/update.py

```diff
@@ -235,62 +235,6 @@ def _get_direct_dependencies_requirements(req_dev: bool) -> set:
 
         return direct_dependencies
 
-    def _get_all_packages_versions(self) -> dict:
-        """Parse Pipfile.lock file and retrieve all packages in the corresponding locked versions."""
-        try:
-            with open("Pipfile.lock") as pipfile_lock:
-                pipfile_lock_content = json.load(pipfile_lock)
-        except Exception as exc:
-            # TODO: open a PR to fix this
-            raise DependencyManagementError(
-                f"Failed to load Pipfile.lock file: {str(exc)}"
-            ) from exc
-
-        result = {}
-
-        for package_name, package_info in pipfile_lock_content["default"].items():
-            if "git" in package_info:
-                self._create_unsupported_package_issue(package_name, "git")
```

Comment:
I think the definition of this method should also be removed with the proposed change, as it is only used within the removed code

https://github.com/thoth-station/kebechet/blob/a7c2f9d6c207cb3740b268fd2502c0bdb6c15bdf/kebechet/managers/update/update.py#L294-L327)

Reply:
ack, thanks